test/08_faucet_tests.ts

```diff
@@ -60,48 +62,46 @@ describe('Testing the Holograph Faucet', async () => {
 
     it('isAllowedToWithdraw(): User is allowed to withdraw for the first time', async function () {
       expect(
-        await FAUCET.functions.isAllowedToWithdraw(DEPLOYER)
+        await FAUCET.isAllowedToWithdraw(DEPLOYER)
```

Comment:
There is no reason to remove this.

Reply:
BTW, there is another benefit of using `functions`, better intellisense in IDE as it displays only the functions without other Ethers Contract instance methods. However, I need to figure out why it gives different result than calling the functions directly.

@ACC01ADE @alexanderattar